CONTRIBUTING.rst

```diff
@@ -60,14 +46,11 @@ on the latest stable version.
 
     $ docker-compose up -d
```

Comment:
Again this is going to start the procrastinate worker, which will fail to start because the database structure has not been created yet.

Reply:
I separated the commands in here as well. :)